docs/src/main/sphinx/release/release-405.md

```diff
@@ -81,20 +86,27 @@
   `timestamp with timezone` data types. ({issue}`15204`)
 * Improve performance of queries that read a small number of columns and queries
   that process tables with large Parquet row groups or ORC stripes. ({issue}`15168`)
+* Improve stability and reduce peak memory requirements when reading from
+  Parquet files. Add the `parquet.max-buffer-size` catalog configuration to
```

Comment:
It's not clear how the configuration property relates to improving stability and memory requirements. Also, that configuration option is not new. It's been around for a long time.

I would just remove the second sentence.